bloom-housing/bloom
# fix: add three new fields to base view

## Issue Overview

This PR addresses #3398 


## Description

The doorway team wants three additional fields to the base view response in /listings. Luckily all of these are on the listings entity so it should not adjust the query.

The fields are 
- "updatedAt"
- "isWaitlistOpen"
- "neighborhood"

## How Can This Be Tested/Reviewed?

This will not change anything on the front-end. You can test this by hitting the /listings endpoint `http://localhost:3100/listings?view=base&limit=all&filter[0][$comparison]==&filter[0][jurisdiction]=eaa2bbe4-8cd1-4c56-883e-d8de6ba1bbcc&filter[1][$comparison]==&filter[1][status]=active&orderBy[0]=mostRecentlyPublished&orderDir[0]=DESC` (you will need to change the jurisdiction id to one that you have seeded). The new fields should be returned in the response

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
